### ISSUE TITLE
Faulty pgn export in the opening builder

### ISSUE THREAD

**Issue by alexandru-duca** [marcusbuffett/chessmadra-frontend#9]
I imported a pgn file with the following text using the opening builder:
`1. e4 e5 (1... d5 2. exd5) 2. Nf3`
Immediately exporting the imported file yields a pgn file with the following content:
`1. e4 e5 (d5 2. exd5) 2. Nf3`

Not only are the files different, but the sidelines of the exported pgn file can't be parsed by traditional pgn readers.

**Comment by marcusbuffett** [comment]
Oh yeah I see the problem, should be an easy fix, thanks for the report

**Comment by alexandru-duca** [comment]
Indeed, it does work better now, but the exported pgn file is still slightly different.
I will go into more detail now (which may be unnecessary) in order to elaborate on my observation.

I went to the [Lichess analysis board](https://lichess.org/analysis).
The following text was displayed in the "PGN" box below the board after entering my moves:
`1. e4 e5 (1... d5 2. exd5) 2. Nf3`

I created a pgn file with the above text in it and imported it in the opening builder.
My moves, which I entered in the Lichess analysis board, were perfectly migrated.
Immediately exporting my repertoire in the opening builder now yields a pgn file with the following content:
`1. e4 e5 (1...d5 2. exd5) 2. Nf3`

Notice that there is one small difference: The space between `1...` and `d5` is missing.
I do not know if this causes trouble later on, so I just wanted you to know this:
Your pgn format and Lichess' pgn format differ slightly (regarding this one detail).
